pkg/stis/src/cs8.c

```diff
@@ -52,7 +54,7 @@ int main (int argc, char **argv) {
 	output = calloc (STIS_LINE+1, sizeof (char));
     addPtr(&ptrReg, output, &free);
 	if (input == NULL || output == NULL) {
-	    printf ("ERROR:  Can't even begin:  out of memory.\n");
+	    printf("ERROR:  Can't even begin:  out of memory.\n");
```

Comment:
Is there a reason why some lines are not replaced?

Reply:
I avoided changing CLI code because I didn't see a point mixing logging with standard output from an entry-point. 

If @stscirij wants me to do those too, I can.